src/Kernel/Variable.class.st

```diff
@@ -72,6 +72,11 @@ Variable >> acceptVisitor: aProgramNodeVisitor node: aNode [
 	^ aProgramNodeVisitor visitVariableNode: aNode
 ]
 
+{ #category : #testing }
+Variable >> allowShadowing [
```

Comment:
it should be allow**s**~ with **s** at the end.

Reply:
fixed, thanks!